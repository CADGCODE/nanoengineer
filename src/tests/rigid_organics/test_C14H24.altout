======= test_C14H24.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= stderr =======
cutover to fine tolerance at 3
== structure comparison ==
error reading atom 0 from test_C14H24.xyzcmp
could not read base positions file from -B<filename>
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_C14H24.mmp  > test_C14H24.xyz
final RMS gradient=2499.722525 after 6 iterations
== structure comparison ==
======= test_C14H24.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_C14H24.mmp
# Output File: test_C14H24.xyz
# Trace File: test_C14H24.trc
# Number of Atoms: 38
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 0 columns:
#
0 3658.13 5201.39
1 2553.83 3002.06
2 2500.66 2920.18
3 2499.74 2918.79
4 2499.72 2918.77
5 2499.72 2918.77
6 2499.72 2918.77
# Done: Minimization final rms: 2499.722525, highForce: 2918.767802
======= test_C14H24.xyz =======
38
RMS=2499.722525
C -0.404177 0.286043 1.283208
C -0.387357 -1.247169 1.100617
C -0.172519 0.969594 -0.080228
C 0.978597 -1.682886 0.530060
C -0.619943 -1.932399 2.474878
C -1.515905 -1.658140 0.129823
C -0.188419 2.512554 0.099083
C -1.298957 0.556939 -1.053270
C 1.190575 0.531998 -0.653270
C 1.205393 -1.000360 -0.833682
C 2.582623 -1.437409 -1.408904
C 0.078501 -1.417295 -1.805196
C -1.287051 -0.976864 -1.233775
C -2.420098 -1.393029 -2.213337
H -1.375554 0.595919 1.681432
H 0.376304 0.587731 1.987557
H 0.998078 -2.770569 0.403018
H 1.776758 -1.393900 1.220214
H -0.596121 -3.028916 2.361088
H -1.599621 -1.634456 2.881443
H 0.163146 -1.628249 3.186973
H -1.522641 -2.745640 -0.001943
H -2.480201 -1.343618 0.537775
H -0.030903 3.017113 -0.864824
H 0.613221 2.819458 0.787159
H -1.158168 2.835361 0.512072
H -1.145772 1.040245 -2.024355
H -2.268288 0.870064 -0.651749
H 1.360722 1.021601 -1.615219
H 1.991982 0.825277 0.026024
H 2.754011 -0.948103 -2.382707
H 2.609384 -2.530273 -1.545402
H 3.391175 -1.144799 -0.720702
H 0.238873 -0.939746 -2.776858
H 0.090597 -2.503296 -1.938525
H -2.264233 -0.921252 -3.194621
H -3.394162 -1.064847 -1.815202
H -2.432849 -2.487678 -2.335653
