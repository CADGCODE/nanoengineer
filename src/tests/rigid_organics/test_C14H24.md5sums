stdout F8A4BCE6098F008BFAA38202A3BABFEE
stderr D41D8CD98F00B204E9800998ECF8427E
test_C14H24.xyz 4651ADC01E8C458248AC3D6292BFDBB9
