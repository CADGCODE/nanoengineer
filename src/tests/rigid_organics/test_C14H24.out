======= test_C14H24.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
0
======= stderr =======
cutover to fine tolerance at 3
== structure comparison ==
